satpy/utils.py

```diff
@@ -476,30 +478,75 @@ def _check_yaml_configs(configs, key):
                         pass
     return diagnostic
 
+def _check_package_version(package_name: str) -> Optional[str]:
+    """Check the version of `package_name`.
 
-def _check_import(module_names):
-    """Import the specified modules and provide status."""
-    diagnostics = {}
-    for module_name in module_names:
-        try:
-            __import__(module_name)
-            res = "ok"
-        except ImportError as err:
-            res = str(err)
-        diagnostics[module_name] = res
-    return diagnostics
+    Args:
+        package_name (str): the distribution package name.
+
+    Returns:
+        the version number if available else `None`.
+    """
+    try:
+        return importlib.metadata.version(package_name)
+    except importlib.metadata.PackageNotFoundError:
+        return None
 
+def show_versions(packages=None):
```

Comment:
Two empty lines required between functions.
```suggestion

def show_versions(packages=None):
```